src/components/App/App.js

```diff
@@ -1,18 +1,21 @@
 import { fetchQuestions } from '../../apiCalls';
 import { useState, useEffect } from 'react';
-import { Route, Switch, Link, NavLink, Redirect } from 'react-router-dom';
+import { Route, Switch, Link, Redirect } from 'react-router-dom';
 import NavBar from '../NavBar/NavBar';
 import Quiz from '../Quiz/Quiz';
 import './App.css';
 
 const App = () => {
   const [ questions, setQuestions ] = useState([])
   const [ error, setError] = useState('')
-  const [ score, setScore ] = useState(0);
+  const [ score, setScore ] = useState(0)
+  const [ step, setStep ] = useState(0)
```

Comment:
I should rename step to factNumber to be more descriptive.

Reply:
I don't see "step" in the code here, but if it's a reference to each of your question facts, then yes I'd say "factNumber" is more descriptive.